documentation/manual/src/main/asciidoc/modules/mongodb.asciidoc

```diff
@@ -3473,6 +3473,55 @@ List<Poem> result = session.createNativeQuery( nativeQuery ).addEntity( Poem.cla
 ----
 ====
 
+[[ogm-mongodb-stored-proc-native]]
+
+==== Server-side JavaScript and stored procedures
+
+In MongoDB, it is possible to call server side javascripts as if it is a stored procedure.
+You can use the existing methods in JPA:
+
+.Calling server side javascript with Hibernate OGM with positional parameters
```

Comment:
same here `server-side JavaScript`

I would drop the `with Hibernate OGM`. It's 2 `with` and, well, we are in the Hibernate OGM documentation :).